File: 485-max-consecutive-ones/max-consecutive-ones.cpp
class Solution 
{
    public:
    int findMaxConsecutiveOnes(vector<int>& nums) 
    {
       int count=0;        
       int maxi=0;
       for(int x=0;x<nums.size();x++)
       {
           if(nums[x]==1)
           {
            count++;
           }
           else
           {
            count=0;
           }
           maxi=max(count,maxi);
       }
       return maxi;
    }
};

File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution 
{
    public int findMaxConsecutiveOnes(int[] nums) 
    {
        int count=0;
        int max=0;
        for(int x=0;x<nums.length;x++)
        {
            if(nums[x]==1)
            {
                count++;
            }
            else
            {
                count=0;
            }
            max=Math.max(max,count);
        }
        return max;
    }
}